sdk/spring-sdk/src/test/scala/kalix/springsdk/impl/RestKalixClientImplSpec.scala

```diff
@@ -62,7 +62,6 @@ class RestKalixClientImplSpec extends AnyWordSpec with Matchers with BeforeAndAf
         restDefCall.fullServiceName shouldBe targetMethod.getService.getFullName
         restDefCall.methodName shouldBe targetMethod.getName
         assertMethodParamsMatch(targetMethod, restDefCall.message)
-        restDefCall.metadata.get("testServiceHeader").get() shouldBe "testName"
```

Comment:
@efgpinto, this verification is not needed anymore because the calls from the Rest client doesn't need to have any extra header.